iam.tf

```
@@ -27,7 +27,7 @@ data "aws_iam_policy_document" "lambda_basic" {
       "logs:PutLogEvents",
     ]
 
-    resources = ["arn:aws:logs:*:*:*"]
+    resources = ["arn:aws:logs:*:*:/aws/lambda/${var.lambda_function_name}*"]
```

I suppose we could also do this if we wanted to be more explicit:
```suggestion
    resources = [
      "arn:aws:logs:*:*:/aws/lambda/${var.lambda_function_name}",
      "arn:aws:logs:*:*:/aws/lambda/${var.lambda_function_name}:log-stream:*",
    ]
```